### PR TITLE
Re-add Missing Controls for Test.Shared.targets

### PR DESCRIPTION
**Pull request by karenbtlai**
## Description

Expander, ImageIcon, and PipsPager control tests were missing after a [bad merge](https://github.com/microsoft/microsoft-ui-xaml/pull/3750/files), so is being re-added here.

### CODE REVIEW

File: test/MUXControls.Test/MUXControls.Test.Shared.targets [hunk 1]
```diff
@@ -40,7 +40,10 @@
   <Import Project="..\..\dev\NumberBox\InteractionTests\NumberBox_InteractionTests.projitems" Label="Shared" Condition="$(FeatureNumberBoxEnabled) == 'true'" />
   <Import Project="..\..\dev\RadialGradientBrush\InteractionTests\RadialGradientBrush_InteractionTests.projitems" Label="Shared" Condition="$(FeatureRadialGradientBrushEnabled) == 'true'" />
   <Import Project="..\..\dev\InfoBar\InteractionTests\InfoBar_InteractionTests.projitems" Label="Shared" Condition="$(FeatureInfoBarEnabled) == 'true'" />
+  <Import Project="$(MSBuildThisFileDirectory)\..\..\dev\Expander\InteractionTests\Expander_InteractionTests.projitems" Label="Shared" Condition="$(FeatureExpanderEnabled) == 'true'" />
```
Comment (kmahone) [inline test/MUXControls.Test/MUXControls.Test.Shared.targets:43]: Fix this path so it matches the rest in this file.

**Review by kmahone — CHANGES_REQUESTED** [review]

File: test/MUXControls.Test/MUXControls.Test.Shared.targets (hunk 1 above)
Comment (karenbtlai) [reply test/MUXControls.Test/MUXControls.Test.Shared.targets]: oops, fixed!

**Pull request merged**